app/presenters/brexit_taxons_presenter.rb

```diff
@@ -0,0 +1,41 @@
+class BrexitTaxonsPresenter
+  FEATURED_TAXONS = %w(
+    /going-and-being-abroad
+    /crime-justice-and-law
+    /transport
+    /environment
+    /business-and-industry
+    /education
+  ).freeze
+
+  def featured_taxons
+    @featured_taxons ||= FEATURED_TAXONS
+      .map { |base_path| Taxon.find(base_path) }
+      .map { |taxon| BrexitTaxonPresenter.new(taxon) }
+  end
+
+  def other_taxons
+    @other_taxons ||= ContentItem.find!('/')
+      .linked_items('level_one_taxons')
+      .reject { |content_item| FEATURED_TAXONS.include?(content_item.base_path) }
```

Comment:
Can we also `reject` `/government` from here please?  It's pretty much going to contain duplicates of everything else.

Reply:
(And not feature it)